bcgov/moh-prime
# PRIME-2167 make Pharmanet ID optional for PRNs

[PRIME-2167](https://bcgovmoh.atlassian.net/browse/PRIME-2167)

How to test:

Create a user enrolment and select 

- **College Licences:** BC College of Nurses and Midwives (BCCNM)
- **Nurse Category:** Registered Nurse/Licensed Graduate Nurse
- **Licence Class issued by the college:** Practicing Registered Nurse

You should see the toggle to provide PharmaNet ID as an option instead of a required field for PharmaNet ID
